### PR TITLE
Update Maaslin2 version to use latest from BioC

### PR DESCRIPTION
**Pull request by ljmciver**
Updating Maaslin2 recipe to latest bioC version.

### CODE REVIEW

**Comment by ljmciver** [comment]
Looking at the errors I see "The recipe is currently blacklisted and will not be built". Is this possibly because the existing version is out of date? If not, let me know what the issue is. Thank you!

Tagging @npavlovikj as you are listed as the maintainer for this recipe. Thanks again!

**Comment by npavlovikj** [comment]
@ljmciver , looks like the recipe was blacklisted due to some unsatisfiable errors, so if you want to try re-building it, please remove it from https://github.com/bioconda/bioconda-recipes/blob/master/build-fail-blacklist in your PR as well.

**Comment by ljmciver** [comment]
@npavlovikj Thank you for the heads up! I made that change and will look into the build errors.

File: recipes/maaslin2/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-  url: https://bitbucket.org/biobakery/maaslin2/get/0.99.12.tar.gz
+  url: https://www.bioconductor.org/packages/release/bioc/src/contrib/Maaslin2_1.16.0.tar.gz
   sha256: {{ sha256 }}
   patches:
     - rpath.patch
 
 build:
-  number: 1
+  run_exports:
+    - {{ pin_subpackage('maaslin2', max_pin="x") }}
```
Comment (martin-g) [inline recipes/maaslin2/meta.yaml:17]:
```suggestion
    - {{ pin_subpackage(name|lower, max_pin="x") }}
```

**Comment by ljmciver** [comment]
@martin-g Thank you for the suggestions. I have committed them in this PR.

**Review by martin-g — APPROVED** [review]

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Comment by npavlovikj** [comment]
@ljmciver , looks like the license file is part of the distribution now, so you can also add:
```
license_file: LICENSE
```
in the `about:` Section in `meta.yaml`.

Also, feel free to add yourself as a recipe maintainer if you would like :)

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Review by npavlovikj — APPROVED** [review]

**Comment by renu-pal** [comment]
@ljmciver I am getting the following error while updating Maaslin2 version to 1.1.6.0 for galaxy. Please let me know if anything can be done from my end.

Screenshot from 2024-05-03 02-06-50

**Comment by npavlovikj** [comment]
Sounds like `r-tmb=1.9.6` should be added in the dependencies.

**Comment by ljmciver** [comment]
@renu-pal Thanks for the heads up! I am updating MaAsLin2 to the latest version in a new PR here: https://github.com/bioconda/bioconda-recipes/pull/49266

@npavlovikj Thanks for the suggestion! I will test that out once I get the tests to run through.

Thanks!

**Pull request merged**